lang/test/org/partiql/lang/syntax/SqlParserDateTimeTests.kt

```diff
@@ -17,7 +17,7 @@ class SqlParserDateTimeTests : SqlParserTestBase() {
 
     private val LOCAL_TIME_ZONE_OFFSET = (ZoneOffset.systemDefault().rules.getOffset(Instant.now()).totalSeconds / 60).toLong()
 
-    data class DateTimeTestCase(val source: String, val block: PartiqlAst.Builder.() -> PartiqlAst.PartiqlAstNode)
+    data class DateTimeTestCase(val source: String, val skipTest: Boolean = false, val block: PartiqlAst.Builder.() -> PartiqlAst.PartiqlAstNode)
     private data class Date(val year: Int, val month: Int, val day: Int)
 
     private val MONTHS_WITH_31_DAYS = listOf(1, 3, 5, 7, 8, 10, 12)
```

Comment:
Changed to `camelCase`. I don't feel so strongly about moving this to top-level type. Because as per the logic above, all the immutable properties (`val`) in the class should go to top-level which is not always appropriate in my opinion. 

The error has come up previously as well. I have fixed it in the `CAST` PR. Nevertheless, I have changed it to `50000`.